pkg/scheduler/apis/config/testing/compatibility_test.go

```diff
@@ -1363,3 +1365,167 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
 		})
 	}
 }
+
+func TestAlgorithmProviderCompatibility(t *testing.T) {
+	// Add serialized versions of scheduler config that exercise available options to ensure compatibility between releases
+	testcases := []struct {
+		name        string
+		provider    string
+		wantPlugins map[string][]config.Plugin
+	}{
+		{
+			name: "No Provider specified",
+			wantPlugins: map[string][]config.Plugin{
+				"PreFilterPlugin": {
```

Comment:
nit: maybe define a `wantDefaultPluigns` variable to reflect that "No Provider" or "Default Provider" should both return the `wantDefaultPluigns` .

Reply:
done.